pom.xml

```diff
@@ -6,7 +6,7 @@
     <groupId>io.github.jelastic</groupId>
     <artifactId>jelastic</artifactId>
     <packaging>jar</packaging>
-    <version>7.2.0-3-SNAPSHOT</version>
+    <version>7.2.0-4-SNAPSHOT</version>
```

Comment:
Also make it a release version, not snapshot. And also edit README.md

Reply:
Changed to release version.